### ISSUE TITLE
envkey-source v2.4.0 - hangs indefinitely on windows

### ISSUE THREAD

**Issue by will-byrne-cardano** [envkey/envkey#83]
## Description
Hi, there seems to be an issue with the envkey-source v2.4.0 on Windows machines. I first spotted this while using the python package v2.4.0 but was also able to reproduce in a Git Bash terminal. 

## Details
When attempting `import envkey` in a python script the process hangs indefinitely. This happens as a result of calling the `fetchenv` function inside the `__init__.py` file. The `fetchenv` function calls a process to be run using envkey-source which is installed with the python package (v2.4.0 in this case).

I was able to reproduce this issue in a Git Bash terminal running the command envkey-source after a fresh install of v2.4.0. The process in the terminal hangs indefinitely. 

In both cases, downgrading to v2.3.0 resolved the issue and process was able to fetch env without this issue occurring. 

## OS Info
If useful, the below details were found by running `systeminfo.exe` in powershell:

```
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19045 N/A Build 19045
System Manufacturer:       Dell Inc.
System Model:              Latitude 7410
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 142 Stepping 12 GenuineIntel ~1803 Mhz
BIOS Version:              Dell Inc. 1.21.1, 22/02/2023
```

Python version = 3.9.12

Appears to not be an issue on Linux because we have some pipelines that run on a Linux OS that are not reporting any issues even though they are using v2.4.0 when running builds and tests.

**Comment by danenania** [comment]
Thanks for the detailed bug report @will-byrne-cardano -- much appreciated! We have one other user experiencing a similar issue, though it doesn't seem to occur in all cases on Windows, and I also hadn't thought to attempt with 2.3.x, so it's very helpful to know that this was introduced somewhere between 2.3.0 and 2.4.0. We're investigating.

**Comment by danenania** [comment]
Fixed in envkey-source 2.4.2 / latest SDK versions
